docs/triggers/event-triggers.md

```diff
@@ -0,0 +1,586 @@
+      "date_created": 1626789600    
+    }  
+  ],  
+  "approving_user": {  // Standard hermes user object    
+    "id": "U123",    
+    "team_id": "T123",    
+    "name": "John Doe",    
+    "display_name": "John Doe",    
+    "real_name": "John Doe",    
+    "timezone": "America/Los_Angeles",    
+    "is_bot": false,  
+  }, 
+}
+```
+
+#### shared_channel_invite_declined
+```json
+"data": {    
+  "event_type": "slack#/events/shared_channel_invite_approved",    
+  "invite": {    
+    "id": "I01354X80CA",    
+    "date_created": 1626876000,    
+    "date_invalid": 1628085600,    
+    "inviting_team": {      
+      "id": "T12345678",      
+      "name": "Corgis",      
+      "icon": {...},      
+      "is_verified": false,      
+      "domain": "corgis",      
+      "date_created": 1480946400    
+    },    
+    "inviting_user": {    // Standard hermes user object    
+      "id" : "U123"    
+      "team_id" : "T123",    
+      "name" : "John Doe",    
+      "display_name" : "John Doe",    
+      "real_name" : "John Doe",    
+      "timezone" : "America/Los_Angeles",    
+      "is_bot" : false,    
+    },    
+    "recipient_email": "golden@doodle.com",    
+    "recipient_user_id": "U87654321"
+  },
+  "channel_id": "C12345678",  
+  "channel_type": "public/private/im/mpim",  
+  "channel_name": "test-slack-connect",  
+  "declining_team_id": "T87654321",  
+  "teams_in_channel": [    
+    {      
+      "id": "T12345678",      
+      "name": "Corgis",      
+      "icon": {...},      
+      "is_verified": false,      
+      "domain": "corgis",      
+      "date_created": 1626789600    
+    }  
+  ],  
+  "declining_user": {  // Standard hermes user object    
+    "id" : "U123",    
+    "team_id" : "T123",    
+    "name" : "John Doe",    
+    "display_name" : "John Doe",    
+    "real_name" : "John Doe",    
+    "timezone" : "America/Los_Angeles",    
+    "is_bot" : false,  
+  }, 
+}
+```
+
+#### shared_channel_invite_received
+```json
+"data": {  
+  "event_type": "slack#/events/shared_channel_invite_received",  
+  "invite": {      
+    "id": "I028YDERZSQ",      
+    "date_created": 1626876000,      
+    "date_invalid": 1628085600,      
+    "inviting_team": {        
+      "id": "T12345678",        
+      "name": "Corgis",        
+      "icon": {...},        
+      "is_verified": false,        
+      "domain": "corgis",        
+      "date_created": 1480946400      
+    },      
+    "inviting_user": {       // Standard hermes user object    
+      "id": "U123",    
+      "team_id": "T123",    
+      "name": "John Doe",    
+      "display_name": "John Doe",    
+      "real_name": "John Doe",    
+      "timezone": "America/Los_Angeles",    
+      "is_bot": false,      
+    },      
+    "recipient_user_id": "U87654321"    
+  },    
+  "channel_id": "C12345678",  
+  "channel_type": "public/private/im/mpim",  
+  "channel_name": "test-slack-connect",  
+}
+```
+
+## Usage
```

Comment:
We should state the assumption of using CLI trigger instead of runtime trigger (and link to that section)